execution-core/src/transfer/phoenix.rs

```diff
@@ -35,6 +35,9 @@ pub use phoenix_core::{
     StealthAddress, TxSkeleton, ViewKey, NOTE_VAL_ENC_SIZE, OUTPUT_NOTES,
 };
 
+/// The minimum gas price
+pub const MINIMUM_GAS_PRICE: u64 = 1;
```

Comment:
this const makes more sense in transfer.rs since it is not only used for phoenix